docs/admin/etcd.md

```diff
@@ -1,49 +1,196 @@
 ---
```

Comment:
I'm not good at naming - I'm fine with that, but will leave that decision to others.